azure-pipelines.yml

```diff
@@ -86,6 +87,43 @@ stages:
                   tags:
                     - ${{ replace(variables['Build.SourceBranch'],'refs/tags/','') }}
 
+  # push to production (canary-* tags)
+  - ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/canary-') }}:
```

Comment:
na canary stack sa to pushne a automaticky releasne len pre canary- tagy. Pre dev- tagy som to nerobil lebo to tam nemate vo zvyku davat https://github.com/keboola/job-queue-command-wrapper/pull/9#discussion_r1975128472